### ISSUE TITLE
RPM Repository

### ISSUE THREAD

**Issue by tommytran732** [google/gvisor#7990]
### Description

Right now gVisor only seems to have an apt repository, it would be great to have a repository for .rpm packages as well.

The `golang-gvisor` on Fedora and EPEL are very outdated and recent builds seem to be failing as well. It would be so much better if we could use an official RPM repo from Google :)

**Comment by dngray** [comment]
They did update it [a while ago](https://koji.fedoraproject.org/koji/packageinfo?packageID=30870). Went from that ancient version `20211129` to `20240408` but it seems its back to failing again with the build.

It would be good if this could provided as an official rpm repository that wasn't frozen along with fedora releases.

**Comment by dreua** [comment]
Just to clear up possible misconception: Fedora doesn't necessarily "freeze" a package for a release, given that there is a maintainer who wants to update the package and assuming those updates can be considered patch level updates (they are not exactly using semantic versioning here 😉 ) I'm not aware of any rules or mechanisms preventing that.

**Comment by dngray** [comment]
That's true, think an official repository (perhaps [COPR](https://docs.fedoraproject.org/en-US/neurofedora/copr/)) still makes sense though as there are frequent enough releases of this and they largely do not depend on anything underlying on the system.

More testing on fedora based distributions would be beneficial and I think this one is holding that up. There are a few fedora users here and no doubt they are reporting bugs with their very old version.
